content/docs/beyond-the-basics/public-attestation/index.md

```diff
@@ -1,14 +1,175 @@
 ---
- title: "Public Assets vs Private Assets"
- description: "Transparency through Public Attestation"
+ title: "Public Attestation"
+ description: "Public Assets vs Private Assets"
  lead: "Transparency through Public Attestation"
  date: 2021-05-18T14:52:25+01:00
  lastmod: 2021-05-18T14:52:25+01:00
- draft: true
+ draft: false
  images: []
  menu:
    docs:
      parent: "beyond-the-basics"
- weight: 41
- toc: false
----
\ No newline at end of file
+ weight: 42
+ toc: true
+---
+
+You may wish to attest information to the general public, without the need for viewers to log-in to their RKVST account. `Public Assets` can be used to publicly assert data, also referred to as Public Attestation. For example, you may attest to data containing a vulnerability report against an OpenSource software package or the maintenance records for a building.
+
+Private Assets can only be shared through the creation of [Access Policies](../../rkvst-basics/sharing-assets-with-obac/). Public Assets, however, may be shared with a read-only `Public URL`, similar to the link sharing you may have seen in file sharing services such as Google Drive or DropBox. 
+
+Any Events updating a Public Asset will also be public, and will each have their own unique Public URL.
+
+Following the link to a Public Asset or Public Event will allow read-only access to its information, without the need to sign in to RKVST.
+
+
+## Creating a Publicly Attested Asset
```

Comment:
It maybe useful to add a reference in `RKVST Basics -> Creating An Asset` pointing here if someone wants to create a public asset?

Reply:
I linked to that section in step one, "1. Create an Asset with your desired attributes. See [Creating an Asset](https://docs.rkvst.com/docs/rkvst-basics/creating-an-asset/) for detailed instructions." but let me know if you think it needs to be more clear.

Reply:
Good idea, I will add that!